CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## Unreleased
+
+## 1.0.0-alpha-0
+
+* Adopt OpenTelemetry 1.0.0(-alpha) (#32)
```

Comment:
I don't know what this sentence means. Is it saying that you're adopting the version format, or that you're depending on OTel v1.0.0 ?

Reply:
It refers to the dependency adoption: https://github.com/open-telemetry/opentelemetry-java-contrib/pull/32.  Clarified a bit if that helps.

Reply:
I guess I'm mostly confused by the usage of the word "adopt" here. Why not just say "Depend on"?